### DISCUSSION TITLE
How can I use a `nofail` alternative with mergerfs?

### DISCUSSION THREAD

**Discussion by ghost** [trapexit/mergerfs#1097] [Q&A] [answered]
Using mergerfs the computer refused to boot with the following error
```
[Failed] Failed to mount /mnt/storage
[Depend] Dependency failed for Local File Systems
You are in emergency mode. After logging in, type “journalctl -xb” to view system logs, “systemctl reboot” to reboot, “systemctl default” or “exit” to boot into default mode.
Root password for maintenance (or press Control-D to continue):
```
I added the `nofail` option and it boots up fine but now I can't mount the folder because I get
```
fuse: unknown option nofail
```

How can I use a `nofail` alternative with fuse? I'm looking for a way more specific than [this](https://unix.stackexchange.com/a/590869/513177).

The error was because the mount point wasn't empty. I may need `nofail` in the future to avoid this same error.

**Comment by trapexit** [comment, marked as answer]
Why not use `nonempty` if that's the concern?
